docs/API.md

```diff
@@ -105,17 +105,7 @@ __Parameters__
 | Param  | Type  | Description  |
 |---|---|---|
 |`bucketName`  | _string_  | Name of the bucket. |
-| `region`  |  _string_ | Default value is us-east-1 Region where the bucket is created. Valid values are listed below: |
-| | |us-east-1 |
```

Comment:
> but maybe the info is deleted because we don't want to keep updating the code as the region info is prone to change.

yes @ebozduman also users can set MINIO_REGION to any string they want. Hence it is better not to list AWS regions here.